com.liferay.blade.eclipse.provider/src/com/liferay/blade/eclipse/provider/JavaFileJDT.java

```diff
@@ -592,4 +594,86 @@ protected File getFile() {
 		return null;
 	}
 
+	private boolean typeMatch(String expectType, String paramType) {
+		boolean match = false;
+
+		if (expectType.equals(paramType)) {
+			match = true;
+		}
+		else if (paramType.equals("null")) {
```

Comment:
if the method as test(Object o), and user called this method in their code like test(null), in our code we should think it is "match".
Actually, if user pass null as parameter and no compile errors happen, that means whatever the  expectType is, we should think it is "match"